metadata_annotation.ipynb

```diff
@@ -0,0 +1,2466 @@
+{
```

Comment:
<p>We generally don't use "you" in pertpy's examples. "Blabla can be annotated". But before that please give more context for a better story. WHY do we want to annotate cell lines? What are cell lines? Why are we showing this?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/pertpy-tutorials/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>